doozerlib/distgit.py

```diff
@@ -299,16 +299,21 @@ def release_is_recent(self, release):
         if not match:
             self.logger.debug("release doesn't match release_regex; treating distgit as stale")
             return False
-        timestamp = [int(f) for f in match.groups()]
+
+        release_dt = datetime(*[int(f) for f in match.groups()])
+        if self.runtime.gitdata.last_modified_dt and self.runtime.gitdata.last_modified_dt > release_dt:
```

Comment:
I may be confused, but this sounds like... if you make any change anywhere in a branch of ocp-build-data, every build on that branch will be considered stale according to that commit, even if it had nothing to do with it?

I'd be perfectly happy to have changes to an individual distgit trigger a rebuild of that distgit. I'd be happy to have changes to select parts of group.yml trigger full rebuilds. I don't want every tweak, advisory update, or typo fix to trigger a full rebuild.

Maybe I'm missing something...

Reply:
Yeah, possibly a blunderbuss when a rifle is necessary. I could probably tweak it to check for any change to group.yml & the image specific metadata. If you think we should limit it to select parts of group.yml, the complexity of the code is probably not worth it.

Reply:
I've dropped the force rebuild commit from the PR.

Reply:
I agree trying to figure out what changed in group.yml (or streams.yml) is probably too hard. I think it's best to leave it to whoever made the central change to rebuild what seems necessary. Does still seem like a good idea to rebuild when the specific metadata files change though.